torch/_inductor/lowering.py

```diff
@@ -2501,7 +2501,7 @@ def tensor(data, *, dtype=None, device=None, layout=None, pin_memory=False):
     if isinstance(_unwrap(data), int):
         dtype = dtype or torch.int64
     else:
-        dtype = dtype or torch.get_default_dtype()
+        dtype = dtype or torch.float64
```

Comment:
@eellison @ezyang I feel it should be safe for us to set the new empty tensor to torch.float64 because this is only for the scalar tensor. If we enable the code here, we may not need the modifications to `fake_impls.py`.

Reply:
Just found this lowering rule is also for `torch.tensor`. will try to pass dtype rather than change the hardcode here.

Reply:
the change here is reverted. pass default dtype when creating fx graph node.